src/Vehicle/Vehicle.h

```diff
@@ -1482,7 +1484,7 @@ private slots:
     uint                _messagesSent;
     uint                _messagesLost;
     uint8_t             _messageSeq;
-    uint8_t             _compID;
+    uint8_t             _mainCompID;
```

Comment:
the naming is maybe a bit unfortunate, for the vehicle's sysid we have _id, and for it's compid we have _mainCompID ... 
maybe streamlining to e.g.

_mainSysID, _mainCompID
_sysID, _compID
_vehicleSysID, vehicleCompID 

or alike could improve code readability